shared/utility/math/filter/ParticleFilter.hpp

```diff
@@ -98,7 +98,8 @@ namespace utility::math::filter {
          */
         ParticleFilter(const StateVec& mean,
                        const StateMat& covariance,
-                       const ResampleMethod& resample_method_ = ResampleMethod::RESIDUAL | ResampleMethod::SYSTEMATIC) {
+                       const ResampleMethod& resample_method_ = ResampleMethod(ResampleMethod::RESIDUAL
+                                                                               | ResampleMethod::SYSTEMATIC)) {
```

Comment:
Trent had a point that this should be a series of constants rather than an enum.

Reply:
Int constants are ugly :p 

see #706